src/distributions/mod.rs

```diff
@@ -174,13 +174,19 @@ fn ziggurat<R: Rng+?Sized, P, Z>(
         // this may be slower than it would be otherwise.)
         // FIXME: investigate/optimise for the above.
```

Comment:
I think you can remove this comment; we're already removing the `next_f64` function from `Rng`.